### PR TITLE
Mech fabricators can now store bluespace mesh

### LINKED ISSUE #24591: Robotics Fabricator bluespace cell.
If RnD researched bluespace cell, they appear in Exosuit Fabricator menu, but there's no way to create them because Exosuit Fabricator doesn't accept bluespace crystals.

### PR DESCRIPTION
**Pull request by Profakos**
Now you can finally manufacture the Bluespace Cells. 

Fixes #24591

Also adds animations for adding bluespace mesh and titanium sheets to the fabricator.

### CODE REVIEW

**Comment by XDTM** [comment]
Won't allow you to insert single crystals, but it's better than nothing.

**Comment by Profakos** [comment]
Oh, i didn't know the Protolathe accepted single crystals. I will probably add that too, for consistency's stake,

**Comment by Profakos** [comment]
Examining code also revealed that the protolathe doesn't check if you are trying to insert cursed undroppable items, but that is a fix for a different PR.

File: code/game/mecha/mech_fabricator.dm [hunk 1]
```diff
@@ -461,8 +452,54 @@
...
+			return 1
+
+		var/inserted = materials.insert_item(W)
+		if(inserted)
+			to_chat(user, "<span class='notice'>You add [W] to the [src].</span>")
+			if(W && W.materials.len)
+				qdel(W)
+				var/mat_overlay = "fab-load-bluespace"
+				add_overlay(mat_overlay)
+				sleep(10)
```
Comment (optimumtact) [inline code/game/mecha/mech_fabricator.dm]: check !qdeleted after this sleep

**Comment by optimumtact** [comment]
Thanks

**Pull request merged**